src/pages/home/release-notes/index.md

```diff
@@ -7,11 +7,21 @@ Keywords:
 
 # Release notes
 
+## January 19, 2024
+
+### Android Campaign Classic 2.1.0
+
+* Added support for out-of-the-box push notifications:
+  * Basic notifications, including remind me later and call-to-action functionality
+  * Manual and automatic image carousels of two to five items
```

Comment:
manual and automatic carousels have a minimum of 2 images but manual filmstrip carousel has a minimum of 3 as there will be a left, center, and right image. additionally, as of now there is no maximum image count.

Reply:
changing to 3 for all versions